### DISCUSSION TITLE
Position covariance drawing in 1.59 release

### DISCUSSION THREAD

**Discussion by IvDmNe** [foxglove/community#761] [Q&A]
Hello! I found a pretty feature in the recent release 1.59 with drawing position covariances as ellipses (https://github.com/foxglove/studio/pull/6281). Now I try to modify my user script that generates ```foxglove.LocationFix``` messages to be able to render covariances, but there is not differences in Map panel: positions are displayed only as points.
What I have done:
1. updated foxglove studio (Help -> About shows 1.59 version);
2. added ```position_covariance_type``` = 3 (COVARIANCE_TYPE_KNOWN) and ```position_covariance``` with diagonal elements equal to 10 (to make sure that covariance is not very small to be visible);
3. Changed message type to ```sensor_msgs/NavSatFix``` (not sure that it changes the way the message is processed for rendering points in Map panel).

Please, can you provide sample message with displayed covariance or tell what am I doing wrong?

**Comment by defunctzombie** [comment]
Here's an example: https://github.com/foxglove/studio/blob/main/packages/studio-base/src/panels/Map/index.stories.tsx#L372

Better yet is if you provide a sample of your data so we can see why that is not working and provide feedback. Without data it is hard to answer such questions.

**Comment by IvDmNe** [comment]
Oh, I found the problem. I expected to see ellipses around all path points while exploring MCAP file, but only for current point the ellipse is shown. Thanks, @defunctzombie for providing example message!
